commons-test/src/main/java/org/infinispan/commons/test/JUnitTestListener.java

```diff
@@ -60,8 +61,10 @@ public void testAssumptionFailure(Failure failure) {
    }
 
    private String testName(Description description) {
+      StringBuilder sb = new StringBuilder();
       String className = description.isSuite() ? "suite" : description.getTestClass().getSimpleName();
-      return className + "." + description.getMethodName();
+      sb.append(className).append(".").append(translateTestName(description));
+      return sb.toString();
```

Comment:
In JDK 11 `+` uses `invokedynamic` and at least in theory can be optimized better than an explicit `StringBuilder`.